docs/source-pytorch/clouds/cluster_advanced.rst

```diff
@@ -88,6 +88,18 @@ To train a model using multiple nodes, do the following:
 
         #SBATCH --signal=SIGUSR1@90
 
+    You can change this signal if your environment requires the use of a different one, for example
```

Comment:
This step by step guide here is to show how to get started quickly with default settings. So I think it's better not to talk about customization here directly. 

I would move this new paragraph to the detail description in the section "Enable auto wall-time resubmitions" below. What do you think?